tests/test_synthesize_morphologies.py

```diff
@@ -387,12 +468,83 @@ def run_with_mpi():
         print(f"============= #{rank}: Checking results")
         expected_size = 18
         assert len(list(iter_morphology_files(tmp_folder))) == expected_size
+        check_yaml(DATA / "apical.yaml", args["out_apical"])
+        check_yaml(DATA / "apical_NRN_sections.yaml", args["out_apical_nrn_sections"])
+    finally:
+        # Clean the directory
+        print(f"============= #{rank}: Cleaning")
+        shutil.rmtree(tmp_folder)
 
-        check_yaml(DATA / ("apical.yaml"), args["out_apical"])
-        check_yaml(
-            DATA / ("apical_NRN_sections.yaml"),
-            args["out_apical_nrn_sections"],
+
+def run_with_mpi_boundary():
```

Comment:
Isn't it simpler to have a parameter to enable / disable the boundary?

Reply:
I removed this test, we don't run it anyway